src/lib/resolveNodeModuleToDenoModule/resolveNodeModuleToDenoModule.ts

```diff
@@ -41,16 +41,16 @@ type Result =
 
 export function resolveNodeModuleToDenoModuleFactory(
     params: {
-        userProvidedPorts: { [nodeModuleName: string]: string };
-        dependencies: { [nodeModuleName: string]: string };
-        devDependencies: { [nodeModuleName: string]: string };
+        userProvidedPorts: Dependencies;
+        dependencies: Dependencies;
+        devDependencies: Dependencies;
```

Comment:
I'm not sure it's worth it to create a type for that its just a `Record<string, string>`.  
I've wrote it like `{ [nodeModuleName: string]: string }` to increace leagability.  
Having the type deported away makes the reader look elswhere...